get_ci_vm/README.md

```diff
@@ -1,10 +1,151 @@
-A container image for use by containers-org repos.
-`hack/get_ci_vm.sh` script.  It should not be
-used via any other mechanism.
+# Overview
+
+This directory contains the source for building [the
+`quay.io/libpod/get_ci_vm:latest` image](https://quay.io/repository/libpod/get_ci_vm?tab=info).
+This image image is used by many containers-org repos. `hack/get_ci_vm.sh` script.
+It is not intended to be called via any other mechanism.
 
 Example build (from repository root):
 
 ```bash
 make get_ci_vm IMG_SFX=latest
 podman tag get_ci_vm:latest quay.io/libpod/get_ci_vm:latest
 ```
+
+# Operational details
+
+The container expects to be called in a specific way, by a script which
+itself needs to behave according to a set of rules (outlined below).
+This bidirectional requirement scheme is necessary to properly support
+differing runtime details and requirements across multiple repositories.
+
+## Downstream repository `.cirrus.yml` requirements
+
+1. *Must* be valid YAML parsable by a **standard** parser.
+1. *Must* not rely on [a `Starlark` script](https://cirrus-ci.org/guide/programming-tasks/).
+1. *Must* not have any conflicting/overlaping task or alias names.  If a matrix
```

Comment:
My eye finds that confusing: stress on *Must*, to me, means affirmation. I'd really prefer stress on both: *Must not*. (Other instances below).

Reply:
Good catch, I did a search/replace to add the italics and missed that case.  I'll fix it.